src/GovUk.Education.ExploreEducationStatistics.Admin/Services/AdminEventRaiserService.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Events;
+using GovUk.Education.ExploreEducationStatistics.Common.Services.EventGrid;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Services;
+
+public interface IAdminEventRaiserService
+{
+    Task OnThemeUpdated(Theme theme);
+}
```

Comment:
We tend to put all interfaces into separate files, one interface per file, and put them in a separate namespace so they don't create clutter. Or at least I think that was our original intention for doing it this way! 

I didn't mention this before given that you were authoring new code in new projects but thought I'd point it out here given that it's in the Admin. In this case you should see lots of interfaces in `Admin.Services.Interfaces`.